app/api/v1/endpoints/insurance.py

```
@@ -0,0 +1,66 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlmodel.ext.asyncio.session import AsyncSession
+from typing import List
+from uuid import UUID
+from app.db.session import get_db
+from app.schemas.insurance import InsuranceSchema, InsuranceCreate, InsuranceUpdate
+from app.crud import insurance as insurance_crud
+from loguru import logger
```

Ruff (0.8.2)

`loguru.logger` imported but unused

Remove unused import: `loguru.logger`

(F401)